AgentDiscoveries-Frontend/app/src/components/app.js

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+
+import Navbar from "./navbar";
+import Login from "./login";
+import LocationReportSearch from "./search-forms/search-location-reports";
+import RegionSummarySearch from "./search-forms/search-region-summaries";
+import LocationReportSubmit from "./submit-forms/submit-location-report";
+import RegionSummarySubmit from "./submit-forms/submit-region-summary";
+
+import {Tabs, Tab} from "react-bootstrap"
+export default class Test extends React.Component {
+    constructor() {

```

Comment:
If you don't do anything in the constructor, then you can leave it out (by default it will just go straight to the super())